client/src/components/EditTodos.js

```diff
@@ -1,21 +1,17 @@
 import React, { useState } from "react";
+import { updateTodo } from "../API/updateTodo";
 
 const EditTodos = ({ todo }) => {
 
     const [description, setDescription] = useState(todo.description);
 
-    const updateTodo = async (e) => {
+    const editTodo = async (e) => {
         e.preventDefault();
-        const body = { description };
         try {
-            const res = await fetch(`http://localhost:3000/todos/${todo.todo_id}`, {
-                method: "PUT",
-                headers: { "Content-Type": "application/json" },
-                body: JSON.stringify(body)
-            });
-
-            console.log(res);
-            window.location = "/";
+            const res = await updateTodo({ id: todo.todo_id, description: description });
+            if (res.status === 200) {
+                window.location = "/";
```

Comment:
Why use window.location ?

Reply:
Previously, there was no proper state management, and Window.location was used to refresh the todo list.
In commit c46f3fc7a9111f1a26e05f47ee0f532cff3aa165, the state management approach was updated.
Window.location has now been removed and used unstated-next for improved state management.